client-v2/src/shared/actions/ArticleFragments.js

```diff
@@ -12,4 +12,47 @@ function articleFragmentsReceived(articleFragments: {
   };
 }
 
-export { articleFragmentsReceived };
+function removeSupportingArticleFragment(
+  id: string,
+  supportingArticleFragmentId: string
+): Action {
+  return {
+    type: 'SHARED/REMOVE_SUPPORTING_ARTICLE_FRAGMENT',
+    payload: {
+      id,
+      supportingArticleFragmentId
+    }
+  };
+}
+
+function addSupportingArticleFragment(
+  id: string,
+  supportingArticleFragmentId: string,
+  index: number
+): Action {
+  return {
+    type: 'SHARED/ADD_SUPPORTING_ARTICLE_FRAGMENT',
+    payload: {
+      id,
+      supportingArticleFragmentId,
+      index
+    }
+  };
+}
+
+function changeArticleGroup(id: string, newGroup: string): Action {
+  return {
+    type: 'SHARED/CHANGE_ARTICLE_GROUP',
```

Comment:
<nitpick>CHANGE_ARTICLE_FRAGMENT_GROUP?</nitpick> :)

Reply:
This is / was a good point and will change.